### ISSUE TITLE
Show relevant dates in the date picker

### ISSUE THREAD

**Issue by fniessink** [ICTU/quality-time#4185]
# User story

As a quality manager I want to see relevant dates in the date picker such as end of sprint, next release date, etc. so I can easily set a relevant technical debt end date.

## Acceptance criteria

- [ ] `role` can `action`

## Assumptions

- Something is present...

## Open issues

- An issue tracker is not necessarily a source for information about sprints and release dates. 

## Out of scope

- Refactoring ...
- Adding ...

# Tasks

### Frontend

- [ ] Add `user interface element`
- [ ] Update `user interface element`

###  Shared python

- [ ] Change the data model
- [ ] Change endpoint `endpoint`
- [ ] Extend the example reports

### External server

- [ ] Change endpoint `endpoint`
- [ ] Add ...

### Collector

- [ ] Change `collector`
- [ ] Add ...

### Notifier 

- [ ] Change `destination`
- [ ] Add ...

### Internal server

- [ ] Change endpoint `endpoint`
- [ ] Add ...

## Feature tests

- [ ] `role` does `action`
- [ ] ...

## Documentation

- [ ] Update changelog
  - [ ] Describe change
  - [ ] Add deployment instructions to changelog
- [ ] Add instruction to user manual
- [ ] Update screenshots
- [ ] Update deployment instructions
- [ ] Update software documentation

## Related issues

- #378
- Blocked by #9796 

Reported by @Sebastiaan127001

**Comment by Sebastiaan127001** [comment]
%VOTE

**Comment by Sebastiaan127001** [comment]
for this we can query Jira to see the start and end of the sprint (this cannot be done though JQL https://jira.atlassian.com/browse/JRACLOUD-72007)

**Comment by Sebastiaan127001** [comment]
relevant dates could also be based on all the set dates in the different metrics of a report, for instance the end date of a calendar date source
